wire-ios-data-model/Source/Model/Conversation/ZMConversation+Predicates.swift

```diff
@@ -169,15 +169,18 @@ extension ZMConversation {
                                                                                                        NSNumber(value: ZMConnectionStatus.ignored.rawValue),
                                                                                                        NSNumber(value: ZMConnectionStatus.cancelled.rawValue)]) // pending connections should be in other list, ignored and cancelled are not displayed
         let predicate1 = NSCompoundPredicate(orPredicateWithSubpredicates: [notAConnection, activeConnection]) // one-to-one conversations and not pending and not ignored connections
+        let oneToOneConversationWithoutMetadata = NSPredicate(format: "%K == %d && %K.@count == 1 && (ALL %K.user.%K == nil)", ZMConversationConversationTypeKey, ZMConversationType.oneOnOne.rawValue, ZMConversationParticipantRolesKey, ZMConversationParticipantRolesKey, #keyPath(ZMUser.name))
+        let noOneToOneConversationWithoutMetadata = NSCompoundPredicate(notPredicateWithSubpredicate: oneToOneConversationWithoutMetadata)
+
```

Comment:
Which conversations should satisfy this predicate? I can't quite read it from the name, but if I read the predicate itself, it appears to me that an group conversation would satisfy the predicate.

Reply:
1:1 conversations from other backend users that we failed to get during login.

Reply:
I introduced a new property `hasIncompleteMetadata` to search conversations without metadata